sinch/domains/voice/models/svaml/actions/actions.py

```diff
@@ -0,0 +1,164 @@
+from dataclasses import dataclass
+from typing import Optional, List, TypedDict
+from sinch.core.models.base_model import SinchRequestBaseModel
+from sinch.domains.voice import Destination, ConferenceDTMFOptions
+
+
+class AnsweringMachineDetection(TypedDict):
+    enabled: bool
+
+
+class CallHeader(TypedDict):
+    key: str
+    value: str
+
+
+@dataclass
+class HangupAction(SinchRequestBaseModel):
+    name: str = "hangup"
+
+
+@dataclass
+class ContinueAction(SinchRequestBaseModel):
+    name: str = "continue"
+
+
+@dataclass
+class ConnectPstnAction(SinchRequestBaseModel):
+    name: str = "connectPstn"
+    number: Optional[str] = None
+    locale: Optional[str] = None
+    max_duration: Optional[int] = None
+    dial_timeout: Optional[int] = None
+    cli: Optional[str] = None
+    suppress_callbacks: Optional[bool] = None
+    dtmf: Optional[str] = None
+    indications: Optional[str] = None
+    amd: Optional[AnsweringMachineDetection] = None
+
+    def as_dict(self):
+        payload = super().as_dict()
+        if payload.get("max_duration"):
+            payload["maxDuration"] = payload.pop("max_duration")
+
+        if payload.get("dial_timeout"):
+            payload["dialTimeout"] = payload.pop("dial_timeout")
+
+        if payload.get("suppress_callbacks"):
+            payload["suppressCallbacks"] = payload.pop("suppress_callbacks")
+
+        return payload
+
+
+@dataclass
+class ConnectMxpAction(SinchRequestBaseModel):
+    name: str = "connectMxp"
+    destination: Optional[Destination] = None
+    call_headers: Optional[List[CallHeader]] = None
+
+    def as_dict(self):
+        payload = super().as_dict()
+        if payload.get("call_headers"):
+            payload["callHeaders"] = payload.pop("call_headers")
+
+        return payload
+
+
+@dataclass
+class Option(SinchRequestBaseModel):
+    dtmf: str
+    action: str
+
+
+@dataclass
+class MenuOption(SinchRequestBaseModel):
+    id: str
+    main_prompt: Optional[str] = None
```

Comment:
How are the fields in this class serialized? As `mainPrompt` or `main_prompt`
I would like to be sure some properties are not sent in a format that the server will not understand

Reply:
I've added missing transformation